pkgs/development/misc/resholve/resholve-utils.nix

```diff
@@ -179,7 +179,7 @@ rec {
     lib.extendDerivation true passthru (stdenv.mkDerivation {
       src = unresholved;
       version = unresholved.version;
-      pname = "resholved-${unresholved.pname}";
+      name = "resholved-${unresholved.name}";
```

Comment:
The `pname`/`version` fix will probably take a while because the issue is in Nix: the JSON nix generates (and is consumed in repology) doesn't include the `pname`/`version` attributes, only the `name` one (and in the cases where `pname`/`version` is set but `name` is not, we actually concat both).

See https://github.com/NixOS/nix/pull/4463.

So I suggest that if you want to fix package names in repology in the short term, just leave the pname untouched, this is how most builders should work anyway.

Reply:
I could flip the distinction over to prefixing the "unresholved" name. I think it's less helpful there, but it meets my meta goal of making the dependency graph easier to read.

Reply:
This looks good, since it follows the same logic as `unwrapped` derivations.

Just it needs to be a suffix instead of prefix.